### ISSUE TITLE
Check mode introduces issue with `molecule test`

### ISSUE THREAD

**Issue by retr0h** [metacloud/molecule#217]
Check mode from #201 introduces a bug with `molecule test` in the vagrant provisioner.  Does not appear to impact docker.

```
[jodewey:~/git/ansible-systems/roles/etc_hosts] [venv] master+ 1 ± rm -rf .molecule
[jodewey:~/git/ansible-systems/roles/etc_hosts] [venv] master+ 1 ± molecule test
ERROR: Unable to find an inventory file, specify one with -i ?
None
```

**Comment by retr0h** [comment]
Assigning to @kireledan.

**Comment by kireledan** [comment]
Oh interesting. I didn't notice any bugs with mine. I'll take a look when I get back from vacation this weekend.

**Comment by retr0h** [comment]
> Oh interesting. I didn't notice any bugs with mine. I'll take a look when I get back from vacation this weekend.

Yeah, it looks to occur when the inventory file has been removed.  So removing the `.molecule/` directory triggers the bug.

**Comment by kireledan** [comment]
I can't seem to reproduce this. I've deleted my .molecule directory, but my ```molecule test``` still works.

**Comment by retr0h** [comment]
> I can't seem to reproduce this. I've deleted my .molecule directory, but my molecule test still works.

Have you tested with a vagrant provider?  I can't reproduce with docker.

**Comment by kireledan** [comment]
Yeah. Im testing with a vagrant provider.

**Comment by retr0h** [comment]
Corrected in #222.
